.evergreen/config/build/_template-build-libmongoc.yml

```diff
@@ -0,0 +1,26 @@
+tasks:
+  # TODO: Enable the following two builds once libmongoc 1.25.0 is released
```

Comment:
Should this comment also refer back to `name: "build-php-libmongoc"` in `build-task-groups.yml`?

And I think there's a related area near the bottom of `test-variants.yml`.

Reply:
Added comments here and in other files to cross-reference usage.